azure-quantum/tests/unit/test_job_results.py

```diff
@@ -86,8 +296,8 @@ def decode(): str
         download_data = DowloadDataMock()
         download_data.decode = Mock(return_value=results_as_json_str)
         job.download_data = Mock(return_value=download_data)
-        
-        return job.get_results()
+
+        return job
```

Comment:
Could this be the cause of the issue? @kikomiss @LawsonGraham

Reply:
not this - see

def _get_job_results(self, output_data_format, results_as_json_str):
        job = self._mock_job(output_data_format, results_as_json_str)
        
        return job.get_results()
    
    def _get_job_results_histogram(self, output_data_format, results_as_json_str):
        job = self._mock_job(output_data_format, results_as_json_str)

        return job.get_results_histogram()
    
    def _get_job_results_shots(self, output_data_format, results_as_json_str):
        job = self._mock_job(output_data_format, results_as_json_str)
        
        return job.get_results_shots()